Source/WebInspectorUI/UserInterface/Views/DOMTreeElement.js

```diff
@@ -2038,6 +2038,10 @@ WI.DOMTreeElement = class DOMTreeElement extends WI.TreeElement
             text = WI.UIString("Event");
             handleClick = this._handleEventBadgeClicked.bind(this);
             break;
+
+        case WI.DOMTreeElement.BadgeType.Scrollable:
```

Comment:
What do we want to do for `handleClick`?  I think it'd be a bit odd for it to purely be presentational when all the other badges do something when clicked.  Perhaps we make it so that clicking toggles `overflow: hidden` (similar to how pressing `H` will toggle `visibility: hidden !important` via `__WebInspectorHideElement__`)?

also ditto ([DOMTreeContentView.js](#diff-10809e8f4fe8e6b42f95bd1c9d63b82b278dae7bf4b52b5b3323e54b6def9dbe):535)